CONTRIBUTING.md

```diff
@@ -64,6 +64,21 @@ To port models from zenodo using Hugging Face hub,
 1. Create a Hugging Face account - https://huggingface.co/
 2. Request to be added to espnet organisation - https://huggingface.co/espnet
 3. Go to `egs2/RECIPE/*/scripts/utils` and run `./upload_models_to_hub.sh "ZENODO_MODEL_NAME"`
+   
+To upload models using Huggingface-cli follow the following steps:
+You can also refer to https://huggingface.co/docs/transformers/model_sharing
+1. Create a Hugging Face account - https://huggingface.co/
+2. Request to be added to espnet organisation - https://huggingface.co/espnet 
+3. Run huggingface-cli login (You can get the token request at this step under setting > Access Tokens > espnet token  
+4. huggingface-cli repo create your-model-name --organization espnet 
+5. git clone https://huggingface.co/username/your-model-name (clone this outside ESPNet to avoid issues as this a git repo)
```

Comment:
I think we already have an automatic script for the repo creation here? https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/asr.sh#L1479

How about writing the notification based on that?

Reply:
Yes that is a very good point . I have updated the notification in asr.sh , please review it I am not an expert in shell